flask_restful/reqparse.py

```diff
@@ -187,19 +190,19 @@ def parse(self, request, bundle_errors=False):
 
                     try:
                         value = self.convert(value, operator)
-                    except Exception as error:
+                    except ValueError as error:
                         if self.ignore:
                             continue
                         return self.handle_validation_error(error, bundle_errors)
+                    except Exception:
+                        if self.ignore:
```

Comment:
That's a good point. And with that dropped, my two added lines that are now 306 and 307 could go away too.